everest-testing/src/everest/testing/core_utils/configuration/libocpp_configuration_helper.py

```diff
@@ -86,3 +100,29 @@ def _set_value_in_v201_config(ocpp_config: json, component_name: str, variable_n
             if (component["name"] == component_name):
                 component["variables"][variable_name]["attributes"][variable_attribute_type] = value
                 return
+
+
+class LibOCPP201ConfigurationHelper(LibOCPPConfigurationHelperBase):
+
+    def _get_default_visitor(self, central_system_port: int | str,
+                             central_system_host: str) -> OCPPConfigAdjustmentVisitor:
+        return _DefaultOCPP201ConfigurationVisitor(central_system_port, central_system_host)
+
+    @staticmethod
+    def create_temporary_ocpp_configuration_db(libocpp_path: Path,
+                                               device_model_schemas_path: Path,
+                                               ocpp_configuration_file: Path,
+                                               target_directory: Path):
+        wd = libocpp_path / "config/v201"
+        subprocess.run(
+            f"python3 {libocpp_path / 'config/v201/init_device_model_db.py'} --out {target_directory / 'device_model_storage.db'} --schemas {device_model_schemas_path}",
+            cwd=wd,
+            check=True,
+            shell=True
+        )
+        subprocess.run(
+            f"python3 {libocpp_path / 'config/v201/insert_device_model_config.py'} --config {ocpp_configuration_file} --db {target_directory / 'device_model_storage.db'}",
+            cwd=wd,
+            check=True,
+            shell=True
+        )
```

Comment:
Question: We only need the `libocpp_path` for these two Python scripts and the component schemas, right?

Reply:
yes

Reply:
That would probably be worth addressing in a further refactor, then. It seems that if we incorporate the Python scripts into everest-testing and allow a marker to specify the locations of the component schemas, the `--libocpp` argument won't be (always) needed anymore.